parsers/jscript.c

```diff
@@ -1402,7 +1402,11 @@ static boolean parseStatement (tokenInfo *const token, tokenInfo *const parent,
 		}
 	}
 
-	copyToken(name, token, TRUE);
+	if ( isType (token, TOKEN_IDENTIFIER) && isKeyword (token, KEYWORD_NONE) )
+		copyToken(name, token, TRUE);
+	else
+		/* Something broken input. Throw away the current token. */
```

Comment:
"Something broken input" does not sound right.
Maybe "Broken input" is sufficient?